src/game/engine/cmdline.cpp

```diff
@@ -23,12 +28,360 @@
+#if defined(CHRONOSHIFT_DEBUG)
+    static const char *_play_codes[] = {
+        "SONY", // 0xE0792D6D
+        "?", // 0x90046ECF
+        "FUNK", // 0xC3EE9A26
+        "SLICK" // 0xED382178
+    };
+
+    static const char *_editor_codes[] = {
+        "BUILD", // 0xA2C09326
+        "MOBIUS", // 0x1F944BB3
+        "CYCLONE", // 0xDE07154D
+        "CARPET", // 0x0E07B213
+        "EDITOR" // 0x16B170B1
+    };
+
+    static const char *_cheat_codes[] = {
+        "JUPITER", // 0xA0E2AB53
+        "NADE", // 0x00532693
+        "PASSWORD", // 0x7DDFF824
+        "CHEATER", // 0x2CB5CF01
+        "BLUB", // 0xB5B63531
+        "?", // 0xDFABC23A
+        "NUKE", // 0x52B19A22
+        "?", // 0xBE79088C
+        "SPOON", // 0xB216AE7E
+        "CARPET" // 0x0E07B213
+    };
+#endif
+
     Special.Init();
+
     Whom = HOUSES_GOODGUY;
+
     g_InMapEditor = false;
+
     DebugUnshroud = false;
+    g_Debug_Playtest = false;
+    //g_Debug_Skip_Intro = false;
+    g_Debug_Flag = false;
+
+    Language = LANGUAGE_ENGLISH;
+
+    // Make sure we have more than one entry stored in argv as the first entry is the program path.
+    if (argc < 2) {
+        return true;
+    }
+
+    DEBUG_LOG("About to parse command line arguments.\n");
+
+    // NOTE: We're starting on 1 because we don't need to know the path of the program, which is stored in argv[0].
+    for (int index = 1; index < argc; ++index) {
+
+        const char *arg = strupr(argv[index]);
+
+        DEBUG_LOG("\tFound: '%s'\n", arg);
+
+        if (!strcasecmp(arg, "/?") || !strcasecmp(arg, "-?") || !strcasecmp(arg, "-h") || !strcasecmp(arg, "/h")) {
+#ifdef PLATFORM_WINDOWS
+            MessageBoxA(nullptr, _help_text, "Chronoshift", MB_ICONINFORMATION);
+#else
+            printf(_help_text);
+#endif
+            return false;
+        }
+
+        // Check for a custom language option.
+        if (strstr(arg, "-L")) {
+
+            const char lang = *((char *)&arg[strlen("-L")]);
+            switch (toupper(lang)) {
+
+                case 'E':
+                default:
+                    Language = LANGUAGE_ENGLISH;
+                    break;
+
+                case 'F':
+                    Language = LANGUAGE_FRENCH;
+                    break;
+
+                case 'G':
+                    Language = LANGUAGE_GERMAN;
+                    break;
+
+                case 'I':
+                    Language = LANGUAGE_ITALIAN;
+                    break;
+
+                case 'S':
+                    Language = LANGUAGE_SPANISH;
+                    break;
+
+                case 'J':
+                    Language = LANGUAGE_JAPANESE;
+                    break;
+
+            };
+
+            continue;
+
+        }
+
+        if (strstr(arg, "-CD")) {
+            const char *path = &arg[strlen("-CD")];
+            CDFileClass::Set_Search_Drives(path);
+            continue;
+        }
+
+        if (strstr(arg, "-DESTNET")) {
```

Comment:
I would remove all the network related stuff for now as we won't even be using IPX due to a lack of support on most modern operating systems.